src/components/view/InfoCard.vue

```diff
@@ -419,7 +419,7 @@
           <div class="resource-detail-item__label">{{ $t('zone') }}</div>
           <div class="resource-detail-item__details">
             <a-icon type="global" />
-            <router-link v-if="!resource.zoneid.includes(',')" :to="{ path: '/zone/' + resource.zoneid }">{{ resource.zonename || resource.zoneid }}</router-link>
+            <router-link v-if="!resource.zoneid.includes(',') && this.isAdmin()" :to="{ path: '/zone/' + resource.zoneid }">{{ resource.zonename || resource.zoneid }}</router-link>
```

Comment:
@shwstppr instead of such role specific checks, we should check if the vue route is routable at all or not?

Reply:
@rhtyd how to check that?
Like for zone, we have a link like `http://localhost:5050/#/zone/00bef679-f1b8-4cfa-95de-c1e9ee098823`. Can we somehow ascertain this will return 404 because zone does actually exist and only logged-in account doesn't have the privilege to see info

Reply:
@shwstppr search and see examples on using $router.resolve to check if the link is routable; something similar perhaps?